cli/commands/version.go

```diff
@@ -1,22 +1,30 @@
 package commands
 
 import (
-	"fmt"
+	"os"
 
 	"github.com/spf13/cobra"
 	"github.com/tarantool/cartridge-cli/cli/version"
 )
 
+var (
+	projectPath      string
+	showRocksVersion bool
+)
+
 func init() {
 	var versionCmd = &cobra.Command{
 		Use:   "version",
-		Short: "Print the version number of Cartridge CLI",
-		Long:  `All software has versions. This is Cartridge CLI's`,
+		Short: "Show version information",
 		Args:  cobra.MaximumNArgs(0),
 		Run: func(cmd *cobra.Command, args []string) {
-			fmt.Println(version.BuildVersionString())
+			if err := version.PrintVersionString(projectPath, cmd.Flags().Changed("project-path"), showRocksVersion); err != nil {
+				os.Exit(1)
```

Comment:
`log.Fatalf(err.Error())` is used for other commands. So, the error shouldn't be logged in `PrintVersionString` function, it's simply returned and printed here.